composer.json

```diff
@@ -20,20 +20,22 @@
             "homepage": "https://github.com/friendsofsymfony/FOSHttpCache/contributors"
         }
     ],
+    "minimum-stability": "dev",
+    "prefer-stable": true,
     "require": {
-        "php": ">=5.4.8",
+        "php": ">=5.5.9",
         "symfony/event-dispatcher": "^2.3||^3.0",
         "symfony/options-resolver": "^2.3||^3.0",
-        "psr/http-message-implementation": "~1.0",
-        "php-http/adapter-implementation": "^0.1.0",
-        "php-http/discovery": "^0.1.1",
-        "php-http/message-decorator": "^0.1.0"
+        "php-http/client-implementation": "^1.0.0",
+        "php-http/discovery": "^0.6.0",
+        "php-http/message": "^0.2.1",
+        "puli/cli": "^1.0.0-beta8"
```

Comment:
Discovery will not work properly if puli/cli isn't installed (whether globally or in the vendors). We should solve this:

1. by making puli/cli a dependency of Discovery (for now)
2. when Puli has become more widespread, remove the dependency and note in the Discovery docs that Puli must be installed globally (as we now docs usually mention Compoer)
3. optionally, make Dependency an optional dependency of FOSHttpCache. 

I'm not sure about 3. On the one hand, it spares people the trouble of some extra deps (in particular, Puli). On the other, requiring Discovery manually is an extra step for our users, who may not really care about this HTTPlug thing as long as it works.